src/map/pc.hpp

```diff
@@ -313,6 +313,7 @@ struct map_session_data {
 		bool cashshop_open;
 		bool sale_open;
 		unsigned int block_action : 10;
+		bool refuse_party;
```

Comment:
I didn't saw the real usage of the var.
Can't this create false positive and refuse all subsequent party request ?